src/test/kotlin/net/maizegenetics/phgv2/cli/AgcCompressTest.kt

```diff
@@ -0,0 +1,158 @@
+package net.maizegenetics.phgv2.cli
+
+import com.github.ajalt.clikt.testing.test
+import org.junit.jupiter.api.AfterAll
+import org.junit.jupiter.api.BeforeAll
+import org.junit.jupiter.api.Test
+import java.io.File
+import java.lang.Exception
+import java.nio.file.Files
+import java.nio.file.Paths
+import java.time.LocalDate
+import java.util.stream.Collectors
+import kotlin.test.assertEquals
+
+class AgcCompressTest {
+    companion object {
+
+        val tempDir = "${System.getProperty("user.home")}/temp/phgv2Tests/tempDir/"
+        @JvmStatic
+        @BeforeAll
+        fun setup() {
+            File(TestExtension.tempDir).mkdirs()
+        }
+
+//        @JvmStatic
```

Comment:
Can this be removed?

Reply:
Actually, it should not be commented out - I did that so I could inspect the files that were created.  I removed the comments.